### PR TITLE
Upgraded information about BYOK

### PR DESCRIPTION
**Pull request by stefanus-heath**
* Removing whitespace
* Adding information about BYOK

### CODE REVIEW

File: content/en/docs/appstore/modules/business-events.md [hunk 1]
```diff
@@ -390,28 +392,34 @@ When this occurs, please execute the following steps:
...
+* `TruststoreLocation` and `TruststorePassword` - [OPTIONAL] The module suports adding a Truststore and password in order to allow for SSL verification of the server.
 
 {{% alert color="warning" %}}Special characters are not allowed in the `BusinessEvents.EventBrokerSpace` constant. {{% /alert %}}
 
 For further explanation on topics and channels, see [Topics and Channels](#topics-channels) and the [Mendix Event Broker](#mendix-event-broker) section.
 
+#### 6.4.2 DevOps tasks not covered when running Own Kafka Cluster
+
+As operating Own Kafka Cluster is outside of the scope of the Mendix Cloud environment, the following `DevOps` tasks should be taken into consideration (this list is not an extensive list of items):
```
Comment (abhishekamte-mx) [inline content/en/docs/appstore/modules/business-events.md]: `Own` need not be captialized. Maybe reword the sentence a bit 
`As operating your own Kafka Cluster is outside of the scope of the Mendix Cloud environment, the following `DevOps` tasks should be taken into consideration (this list is not extensive):`

File: content/en/docs/appstore/modules/business-events.md [hunk 2]
```diff
@@ -390,28 +392,34 @@ When this occurs, please execute the following steps:
...
+
+* Client username and password provision on Kafka - The creation of usernames and password on the Kafka cluster will need to be managed but the customer
+* Topic creation on Kafka - Unless the Kafka cluster is configured with `auto.create.topics.enable` set to true, topics will need to be created by the customer.
```
Comment (abhishekamte-mx) [inline content/en/docs/appstore/modules/business-events.md]: maybe add a link to the topic creation logic subsection

File: content/en/docs/appstore/modules/business-events.md [hunk 3]
```diff
@@ -390,28 +392,34 @@ When this occurs, please execute the following steps:
...
+* Access Control - Unless the Kafka cluster is configured with `allow.everyone.if.no.acl.found` set to true, the ACLs need to be maintained by the customer
```
Comment (abhishekamte-mx) [inline content/en/docs/appstore/modules/business-events.md]: maybe add a word as default `set to true (default setting in Apache Kafka)`

File: content/en/docs/appstore/modules/business-events.md [hunk 4]
```diff
@@ -450,7 +458,7 @@ Here is an example of postgres service that you can add to your `docker-compose.
 
 2. Can I publish my own events from other software directly to a Kafka topic?
 
-    No, that is currently not supported. 
+    No, that is currently not supported when using Mendix Cloud Event Broker.  This can be achieved by using Own Kafka Cluster.
```
Comment (abhishekamte-mx) [inline content/en/docs/appstore/modules/business-events.md]: rather say BYOK and link to it

File: content/en/docs/appstore/modules/business-events.md [hunk 5]
```diff
@@ -170,17 +170,17 @@ In the **Attributes** section, click **Add** to define attributes. Changes you m
...
 | Subscribe to events | Publishing events | Subscribing to events | When publishing, **PublishedBusinessEvent** entity and handler microflow <br>If subscribing, a **ConsumedBusinessEvent** entity |
-| Publish events and<br>Subscribe to events | [Nothing required: if apps can do both, there is no obligation for the service to implement anything] | Publishing events and/or subscribing to events | If no service implementations are selected, then nothing created <br>If publishing, **PublishedBusinessEvent** entity and handler microflow <br>If subscribing, a **ConsumedBusinessEvent** entity <br>If both, then both entities and the handler microflow are created 
+| Publish events and<br>Subscribe to events | [Nothing required: if apps can do both, there is no obligation for the service to implement anything] | Publishing events and/or subscribing to events | If no service implementations are selected, then nothing created <br>If publishing, **PublishedBusinessEvent** entity and handler microflow <br>If subscribing, a **ConsumedBusinessEvent** entity <br>If both, then both entities and the handler microflow are created
```
Comment (abhishekamte-mx) [inline content/en/docs/appstore/modules/business-events.md:181]: @stefanus-heath  i cant see to see the topic resolution documentation, where is it?

**Review by abhishekamte-mx — CHANGES_REQUESTED** [review]

File: content/en/docs/appstore/modules/business-events.md (hunk 5 above)
Comment (stefanus-heath) [reply content/en/docs/appstore/modules/business-events.md:181]: ah, I see the circular reference to nothingness ... will fix

**Comment by Adam-Dupaski** [comment]
@quinntracy please approve this PR if it's ready. Also, there are conflicts that need to be resolved.

**Review by quinntracy — APPROVED** [review]
